os/drivers/video/video_null_driver.c

```diff
@@ -429,18 +533,18 @@ int video_null_init_transfer(FAR null_priv_t *priv)
 	snprintf(filename, FILE_NAMELEN, FILE_FORMAT, priv->cur_frame->wWidth, priv->cur_frame->wHeight, priv->cur_format->description);
 	videodbg("Video File Name: %s\n", filename);
 
-#if 0
 	fd = open(filename, O_RDOK);
 	if (fd < 0) {
 		return -ENOENT;
 	}
 
-	ret = read(fd, priv->trfbuff, priv->reqsize);
+	ret = read(fd, priv->trfbuff, priv->trfsize);
```

Comment:
if (ret != priv->trfsize)

Reply:
We need not confirm that `ret == priv->trfsize` since the number of bytes may very well be less than priv->trfsize.
trfsize calculated as 2 x height x frame (eg. for VGA Frames: - 2 x 320 x 240) is the maximum size of the frame that may be read. Frame size may be smaller than this for certain formats.